Project-2/README.md

```diff
@@ -29,9 +29,78 @@ Tip #4: Work together _as a team_.
   
 2. Use your axioms from question 1 as the basis of an ALCI T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a model of K. This may be graphical or symbolic or both.
 
+TBox:{
```

Comment:
Oh, I see. So this is where the actual review comment is supposed to be formally. My apologies! 

Pending the changes on the comments I have made, which have since been addressed, I approve your Project #2! 

Nice job, Delaney! 

Best, 
Jaron